readme.md

```diff
@@ -59,6 +61,24 @@ There are two options for installing KBCT
   ./target/release/kbct --help
   ```
 
+#### Arch Linux
```

Comment:
Please make this block with the same styling as the previous two items in the list (or change the previous two to fit your styling if you think it's better :smile: ).